documentation/commands.md

```diff
@@ -1,233 +1,243 @@
+# Commands
+
 Commands in the form of JSON messages are used to start and stop file writing.
-Commands can be send through Kafka via the broker and topic specified by the
-`--command-uri` option.  Commands can also be given in the configuration
-file specified by `--config-file <file.json>` (see [commands in config file](#commands-can-be-given-in-the-configuration-file-as-well)).
 
-In the command, the `nexus_structure` defines the HDF hierarchy.
-The `nexus_structure` represents the HDF root object.  The following example
-shows how the HDF tree can be constructed using `children`.
-A child of type `stream` is a marker that a `HDFWriterModule` will insert
-the data into from a Kafka stream at that point in the hierarchy.  The options under
-the key `stream` specify the stream details containing, at least, the topic, the source name and
-the `HDFWriterModule` which should be used for writing.
-Depending on the `HDFWriterModule`, there will be more options specific to the
-`HDFWriterModule`.
+Commands are generally sent through Kafka via the broker and topic specified by the
+`--command-uri` option; however, commands can also be given in the [configuration
+file](Commands via the configuration file).
 
-#### Command to start writing a file
+## Command to start writing
 
-Further documentation:
+The start command consists of a number of parameters which are defined as key-value pairs in the JSON.
+These are:
 
-- [Groups](docs/groups.md)
-- [~~Datasets~~ documentation not yet written]()
-- [Attributes](docs/attributes.md)
-- [~~File Attributes~~ documentation not yet written]()
-- [~~Streams~~ documentation not yet written]()
+- cmd: The command name, must be `FileWriter_new`.
+- job_id: A unique identifier for the request, a UUID for example.
+- broker: The Kafka broker to use for data.
+- start_time: The start time in milliseconds (UNIX epoch) for data to be written from. Optional, if not supplied then the Kafka message time is used.
```

Comment:
```suggestion
- start_time: The start time in milliseconds (UNIX epoch) for data to be written from. Optional, if not supplied then the timestamp of the Kafka message containing the start command is used.
```
Note: How does this work if the start command is supplied via JSON-file?

Reply:
It uses the system clock